controller/controller.go

```diff
@@ -15,94 +14,83 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-// MSTranslateServer specifies the remote MS translate server used by
+// LnxEndpoint specifies the remote Lnx translate server used by
 // brave-core, and it can be set to a mock server during testing.
-var MSTranslateServer = "https://api.cognitive.microsofttranslator.com"
-
-// GoogleTranslateServerProxy specifies the proxy server for requesting
-// resource from google translate server, and it can be set to a mock server
-// during testing.
-var GoogleTranslateServerProxy = "https://translate.brave.com"
-
-const (
-	// GoogleTranslateServer specifies the remote google translate server.
-	GoogleTranslateServer = "https://translate.googleapis.com"
-
-	// GStaticServerProxy specifies the proxy server for requesting resource
-	// from google gstatic server.
-	GStaticServerProxy = "https://translate-static.brave.com"
-
-	languageEndpoint = "/languages?api-version=3.0&scope=translation"
-)
+var LnxEndpoint = os.Getenv("LNX_HOST")
+var LnxApiKey = os.Getenv("LNX_API_KEY")
+var languagePath = "/get-languages"
+var translatePath = "/translate"
 
 // TranslateRouter add routers for translate requests and translate script
 // requests.
 func TranslateRouter() chi.Router {
 	r := chi.NewRouter()
 
 	r.Post("/translate", Translate)
+	r.Options("/translate", HandleCorsPreflight)
 	r.Get("/language", GetLanguageList)
```

Comment:
Currently we use the chromium standard path to get the list: `/translate_a/l`
https://chromium.googlesource.com/chromium/src/+/refs/heads/main/components/translate/core/browser/translate_language_list.cc#152
Could you please change and verify that it works?